Crash Report 3.1.2: NPE CoreMainActivity.hideTabSwitcher mainMenu

**Describe the bug**
Null Pointer Exception, OnActivityResult occurring before onCreateOptionsMenu

**Expected behavior**
no crash

**Logs**
```
Caused by: java.lang.NullPointerException: 

 
  at org.kiwix.kiwixmobile.core.main.CoreMainActivity.hideTabSwitcher (CoreMainActivity.java:507)

 
  at org.kiwix.kiwixmobile.core.main.CoreMainActivity.onActivityResult (CoreMainActivity.java:1356)

 
  at android.app.Activity.dispatchActivityResult (Activity.java:7759)

 
  at android.app.ActivityThread.deliverResults (ActivityThread.java:4598)

 
  at android.app.ActivityThread.handleSendResult (ActivityThread.java:4647)

 
  at android.app.servertransaction.ActivityResultItem.execute (ActivityResultItem.java:49)

 
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)

 
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)

 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1948)

 
  at android.os.Handler.dispatchMessage (Handler.java:106)

 
  at android.os.Looper.loop (Looper.java:214)

 
  at android.app.ActivityThread.main (ActivityThread.java:7045)

 
  at java.lang.reflect.Method.invoke (Method.java)

 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)

 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:964)
```

--- a/core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java
+++ b/core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java
@@ -504,7 +504,9 @@ public abstract class CoreMainActivity extends BaseActivity
       tabSwitcherRoot.setVisibility(View.GONE);
       progressBar.setVisibility(View.VISIBLE);
       contentFrame.setVisibility(View.VISIBLE);
-      mainMenu.showWebViewOptions(!urlIsInvalid());
+      if (mainMenu != null) {
+        mainMenu.showWebViewOptions(!urlIsInvalid());
+      }
     }
   }
 

